### ISSUE TITLE
Allow the bot to join other organization/tenant's meetings

### ISSUE THREAD

**Issue by gabdelgadoawsys** [microsoftgraph/microsoft-graph-comms-samples#426]
I am able to make the bot work with Microsoft Teams meeting invites created under my tenant. But whenever I try to have the bot join Teams meetings created under other tenants, I get the following error:

`Microsoft.IdentityModel.Clients.ActiveDirectory.AdalServiceException: AADSTS700016: Application with identifier <APP ID> was not found in the directory <TENANT ID>.`

Current Setup
1. Tenant A contains the fully configured Azure Bot Service.
2. Tenant A has no 365 subscription (including Microsoft Teams)
3. Tenant B has the 365 subscription (including Microsoft Teams).
4. Tenant B has no Azure subscription.

Is it possible to have the bot (in Tenant A) join meetings created under Tenant B without having to configure a Bot Service in their end?

**Comment by itisslizz** [comment]
Yeah you can. Tenant B just needs to grant the necessary permissions.
https://login.microsoftonline.com/<TENANT B ID>/adminConsent?client_id=<APP ID>

**Comment by muradhaider5786** [comment]
> Yeah you can. Tenant B just needs to grant the necessary permissions.` https://login.microsoftonline.com/<TENANT B ID>/adminConsent?client_id=<APP ID>`

How will it work when Tenant B is not having Azure subscription hence no application with specified id.

**Comment by 1fabi0** [comment]
Your application is multi tenant capable by doing `https://login.microsoftonline.com/<TENANT B ID>/adminConsent?client_id=<APP ID>` which can be shorted to automatically choose tenant id `https://login.microsoftonline.com/common/adminConsent?client_id=<APP ID>` or for organizations only `https://login.microsoftonline.com/organizations/adminConsent?client_id=<APP ID>` and then Tenant B will use the same App Registration on Tenant A since Tenant B admin allowed to use this app Registration

**Comment by gabdelgadoawsys** [comment]
Hello @1fabi0 . The application was granted admin consent on Tenant B's side. The error message in my initial post is now gone. Thank you. 

But there's another error message appeared:
```
Status Code: Forbidden
Microsoft.Graph.Communications.Core.Exceptions.ServiceException: Code: UnknownError
Message: {"errorCode":"7503","message":"Application is not registered in our store.","instanceAnnotations":[]}
Scenario Id: 4e4815b3-05c7-4e08-a0a2-0c9064289a05
```

Any chance you've also encountered this and familiar with this? Thank you.

**Comment by erwinkramer** [comment]
Adding the tenantid param solved this for me: https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.cloudcommunications/new-mgcommunicationcall?view=graph-powershell-1.0#-tenantid
